sdk/avs/Azure.ResourceManager.Avs/api/Azure.ResourceManager.Avs.netstandard2.0.cs

```diff
@@ -1086,6 +1112,12 @@ public partial class CommonClusterProperties
         public System.Collections.Generic.IList<string> Hosts { get { throw null; } }
         public Azure.ResourceManager.Avs.Models.ClusterProvisioningState? ProvisioningState { get { throw null; } }
     }
+    public partial class CustomerManagedEncryption
+    {
+        public CustomerManagedEncryption() { }
+        public Azure.ResourceManager.Avs.Models.EncryptionKeyVaultProperties KeyVaultProperties { get { throw null; } set { } }
+        public Azure.ResourceManager.Avs.Models.EncryptionState? Status { get { throw null; } set { } }
+    }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
     public readonly partial struct DatastoreProvisioningState : System.IEquatable<Azure.ResourceManager.Avs.Models.DatastoreProvisioningState>
```

Comment:
Same here

Reply:
Renamed to `AvsPrivateCloudDatastoreProvisioningState`